spec/lcm/slow/m_to_n_spec.rb

```diff
@@ -0,0 +1,85 @@
+require_relative 'support/connection_helper'
+require_relative 'support/configuration_helper'
+require_relative 'support/s3_helper'
+
+require_relative 'support/project_helper'
+require_relative 'shared_examples_for_synchronization_bricks'
+
+require 'active_support'
+require 'active_support/core_ext'
+
+describe 'Contains M to N relation' do
+  before(:all) do
+    @config = {
+      verify_ssl: false,
+      environment: 'TESTING',
+      master_prefix: 'Insurance Demo Master'
+    }
+    @suffix = ConfigurationHelper.suffix
+    @release_table_name = "LCM_RELEASE_#{@suffix}"
+    @config.merge!(LcmConnectionHelper.environment)
+
+    @rest_client = LcmConnectionHelper.development_server_connection
+
+    @ads = ConfigurationHelper.create_development_datawarehouse(client: @rest_client,
+                                            title: 'Development ADS',
+                                            auth_token: @config[:dev_token])
+
+    domain = @rest_client.domain(@config[:dev_organization])
+    domain.data_products(:all).each { |d| d.delete(force: true) }
+
+    project_helper = ConfigurationHelper.ensure_development_project(
```

Comment:
Similar code found in shared_contexts_for_lcm_specs.rb:114, m_to_n_spec.rb:31 (mass = 40)